src/main/java/com/vaadin/flow/ai/formfiller/FormFiller.java

```diff
@@ -61,6 +61,11 @@ public class FormFiller {
 
     private static final Logger logger = LoggerFactory.getLogger(FormFiller.class);
 
+    static {
```

Comment:
I am just thinking that should we add this to the constructor which is always called in line 122 here:

```
 public FormFiller(Component target, HashMap<Component, String> componentInstructions, ArrayList<String> contextInstructions, LLMService llmService) {
        this.llmService = llmService;
        this.target = target;
        this.componentInstructions = componentInstructions;
        this.contextInstructions = contextInstructions;
        reportUserStatistics();
    }
```

Maybe it is a bit more common/used place for logging. In the end, I believe both have benefits and disadvantages.
If we put it here this will be only logged once, while if we put it to the constructor then every constructor call will produce new log messages, and potentially spam it, ok let's use this approach :).